6-anthos-install/README.md

```diff
@@ -0,0 +1,353 @@
+# Deploying Bank of Anthos 
+These instructions need to be run on the bastion host. To access the bastion host, open the Google Cloud console, and navigate to Compute Engine in the GKE project. In my instance, `prj-bu1-d-boa-gke` Then, select the `gce-bastion-us-west1-b-01` and click on `ssh`. You need to be a whitelisted member to access this node.
+
+
+## Insall required tools
+```console
+sudo su
+yum install git
+yum install google-cloud-sdk-kpt
+yum install jq -y
+sudo yum install kubectl
```

Comment:
wondering why this has sudo if we do `sudo su` first.

Reply:
It's left by mistake